ethsigner/signer/multikey/src/test/java/tech/pegasys/ethsigner/signer/multikey/SigningMetadataTomlConfigLoaderTest.java

```diff
@@ -272,12 +273,34 @@ void hashicorpConfigIsLoadedIfHashicorpMetadataFileInDirectory() {
     final HashicorpSigningMetadataFile metadataFile =
         (HashicorpSigningMetadataFile) metadataFiles.toArray()[0];
 
-    assertThat(metadataFile.getConfig().getSigningKeyPath()).isEqualTo("/path/to/key");
-    assertThat(metadataFile.getConfig().getHost()).isEqualTo("Host");
-    assertThat(metadataFile.getConfig().getPort()).isEqualTo(9999);
-    assertThat(metadataFile.getConfig().getAuthFilePath().toString())
-        .isEqualTo("/path/to/auth-file");
-    assertThat(metadataFile.getConfig().getTimeout()).isEqualTo(50);
+    final HashicorpConfig hashicorpConfig = metadataFile.getConfig();
+    assertThat(hashicorpConfig.getSigningKeyPath()).isEqualTo("/path/to/key");
+    assertThat(hashicorpConfig.getHost()).isEqualTo("Host");
+    assertThat(hashicorpConfig.getPort()).isEqualTo(9999);
+    assertThat(hashicorpConfig.getAuthFilePath().toString()).isEqualTo("/path/to/auth-file");
+    assertThat(hashicorpConfig.getTimeout()).isEqualTo(50);
+  }
+
+  @Test
+  void hashicorpConfigIsLoadedWithTlsDisabled() {
```

Comment:
nit: probably lots of tests required here to show the various valid files:
* tls on with no trust
* tls on with Trust
* tls off, but with Trust

Reply:
Hmm .... probably yes...

Reply:
I have implemented these test cases and refactored hashicorp tests in different file as well. Resolving convo.